app/models/concerns/client_data_type.rb

```diff
@@ -1,14 +1,14 @@
-module ProposalDelegate
+module ClientDataType
```

Comment:
I feel like `Mixin` and `Concern` are good at communicating the purpose of this module, but it's also a bit repetitive - like naming a class `WorkOrderClass`. The example I see for ActiveModel Concerns is always `Taggable` -- trying to think of names that are in that spirit

What about just calling it `ClientData` ?